gostream/standalone_stream_server.go

```diff
@@ -1,247 +0,0 @@
-package gostream
-
-import (
```

Comment:
This entire file was really just an example server that got pulled in with the gostream migration into the RDK. There's no dependency on this from any of RDK/goutils/app. I asked around a couple weeks ago if something else is magically dependent on this and got nothing.